platform-server-kernel/src/main/java/uno/cod/platform/server/core/repository/TeamInvitationRepository.java

```diff
@@ -13,8 +13,8 @@
 
 public interface TeamInvitationRepository extends JpaRepository<TeamInvitation, TeamUserKey> {
     @Query("SELECT DISTINCT invitation FROM TeamInvitation invitation " +
-            "WHERE invitation.key.user.id = :userId")
-    List<TeamInvitation> findAllByUserId(@Param("userId") UUID userId);
+            "WHERE invitation.key.team.enabled = true AND invitation.key.user.id = :userId")
```

Comment:
Ping @Exordian. If the method is called `...AndTeamEnabled` won't this automatically add a constraint to the query?